substrate/client/cli/src/error.rs

```diff
@@ -78,6 +80,16 @@ pub enum Error {
 
 	#[error(transparent)]
 	GlobalLoggerError(#[from] sc_tracing::logging::Error),
+
+	#[error(
+		"Starting an authorithy without network key in {0}.
+		\n This is not a safe operation because the old identity still lives in the dht for 36 hours.
+		\n Because of it your node might suffer from not being properly connected to other nodes for validation purposes.
+		\n If it is the first time running your node you could use one of the following methods.
+		\n 1. Pass --unsafe-force-node-key-generation and make sure you remove it for subsequent node restarts
+		\n 2. Separately generate the key with: polkadot key generate-node-key --file <YOUR_PATH_TO_NODE_KEY>"
```

Comment:
May be instead of `--file` argument recommend using `--base-path` & `--default-base-path`? This seems to be less error-prone and won't require first time users to know the exact network key location.

Reply:
Should be better now!

Reply:
Thanks!